Formula/yadm.rb

```diff
@@ -1,21 +1,22 @@
 class Yadm < Formula
   desc "Yet Another Dotfiles Manager"
   homepage "https://yadm.io/"
-  url "https://github.com/TheLocehiliosan/yadm/archive/2.5.0.tar.gz"
-  sha256 "d022d118a1a477e09afa00c80e10bd94b150d99709e57f01ba48ca2eaaeeb2de"
-  license "GPL-3.0"
+  url "https://github.com/TheLocehiliosan/yadm/archive/3.0.0.tar.gz"
+  sha256 "af968c815817e9de85b60dc9c9a7e6159ed34e4f91ea78cadbcd6c73a0301c06"
+  license "GPL-3.0-only"
```

Comment:
Are you sure about this? A number of their headers say they can be licensed under any later version of GPL: https://github.com/TheLocehiliosan/yadm/search?q=license

Reply:
Sure, my mistake. I've updated the license line. 👍